### DISCUSSION TITLE
Open new tab programmatically

### DISCUSSION THREAD

**Discussion by Abusayid693** [vercel/next.js#36834] [Help]
Anyway to open new tab programmatically preserving all states, Using `window.open("/..", "_blank")` clears all states.

**Comment by icyJoseph** [comment]
~~Oh, this brings back memories, I once found this article: https://medium.com/hackernoon/using-a-react-16-portal-to-do-something-cool-2a2d627b0202, and then later on tried to actually do it. I ran into some issues with styles, but it sorta worked... though, it wasn't something I'd want to use in production.~~ 

~~I haven't tried it with Next.js, if you have the time to hack with it, I'd love to see if you get it to work.~~

### Edit

Just saw you want to use `/..` I got carried away :( I think this won't cut it... and I am not sure it is possible at all, without some localStorage/indexedDb handling (serialise state).

**Comment by Abusayid693** [comment]
@icyJoseph Yeah this seems impossible without using some kind of cookies or it need to be stateless to open new tab.
